msgspec/__init__.pyi

```diff
@@ -41,7 +41,7 @@ class Struct(metaclass=__StructMeta):
         tag: Union[None, bool, str, int, Callable[[str], Union[str, int]]] = None,
         tag_field: Union[None, str] = None,
         rename: Union[
-            None, Literal["lower", "upper", "camel", "pascal"], Callable[[str], str]
+            None, Literal["lower", "upper", "kebap", "camel", "pascal"], Callable[[str], str]
```

Comment:
TIL some people spell this "kebap" instead of "kebab". Since other tools like serde use the "kebab" spelling (https://serde.rs/container-attrs.html#rename_all), I think we should go with that one.

Reply:
Ah, it seems like kebap is a bit more common in Germany as this is the spelling I see at my local Döner Kebap :-)

But I have absolutely nothing against naming it kebab, especially if this is more common